### DISCUSSION TITLE
"read" command missing?

### DISCUSSION THREAD

**Discussion by kaspar030** [helix-editor/helix#5802] [Q&A] [answered]
Hi,

I'm not finding a "read" command, that inserts the contents of a file. Is there something equivalent in Helix?

Thanks!

**Comment by kaspar030** [comment, marked as answer]
`:insert-output cat <file>` works. (feels a bit awkward, but hey).

**Reply by askreet** [reply]
You can also use the `!` binding to get a quick prompt to this action, making the keystrokes `!cat <file><cr>`. It's rather quite nice once you get used to it!

**Comment by the-mikedavis** [comment]
There's an issue and a PR for `:read`: https://github.com/helix-editor/helix/issues/3796
